Source/Particles/MultiParticleContainer.H

```diff
@@ -127,6 +127,29 @@ public:
     */
     std::unique_ptr<amrex::MultiFab> GetZeroChargeDensity(const int lev);
 
+    /**
+     * \brief Deposit charge density.
+     *
+     * \param[in,out] rho vector of charge densities (one pointer to MultiFab per mesh refinement level)
+     * \param[in] relative_t relative time of deposition
+     * \param[in] icomp component of the MultiFab where rho is deposited (old, new)
+     */
+    void
+    DepositCharge (amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
+                   amrex::Real relative_t, const int icomp = 0);
```

Comment:
let's pass the `relative_t` as const here and below:
```suggestion
                   const amrex::Real relative_t, const int icomp = 0);
```

Reply:
All `const` qualifiers added in https://github.com/ECP-WarpX/WarpX/pull/1828/commits/aa27403a65fa36eac1c37a1e356cb72b8b1510e0.